tests/tests/RDMResponderTest.cpp

```diff
@@ -95,6 +95,100 @@ int ClearSensors(const RDMHeader *header,
   return 0;
 }
 
+// Sensors
+enum { NUMBER_OF_SENSORS = 2 };
+
+const char SENSOR_NAME1[] = "Temperature";
+const char SENSOR_NAME2[] = "Accel";
```

Comment:
We could make this Acceleration and it would still be the same length as sensor 1. Although if I'm reading this, it's only used in the tests, it's not live in Ja Rule?